site-src/implementations.md

```diff
@@ -392,6 +394,21 @@ If you have any suggestions or experience issues with NGINX Gateway Fabric, plea
 [nginx-disc-new]:https://github.com/nginxinc/nginx-gateway-fabric/discussions/new
 [nginx-slack]:https://nginxcommunity.slack.com/channels/nginx-gateway-fabric
 
+
+### ngrok Kubernetes Operator
+
+[ngrok Kubernetes Operator][ngrok-k8s-operator] provides an implementation of the Gateway API that uses [ngrok's ingress-as-a-service][ngrok]. This project uses the Gateway API to support routing traffic from ngrok's global network to applications running on Kubernetes clusters. This easily adds the benefits of ngrok, like security, network policy, and a global presence with the simplicity of cloud service. The operator contains both a Gateway API implementation as well as a controller using Kubernetes Ingress. The Gateway API implementation is currently under development and supports only the Gateway, GatewayClass and HTTPRoute. As the TLSRoute and TCPRoute move from experimental to stable, they will also be implemented.
+
+You can read our [docs][ngrok-k8s-gwapi-docs] or [quickstart guide][ngrok-k8s-gwapi-qs] for more information. If you have any feature requests or bug reports, please [create an issue][ngrok-issue-new]. You can also reach out for help on [Slack][ngrok-slack]
+
+
+[ngrok-k8s-operator]:https://github.com/ngrok/kubernetes-ingress-controller
+[ngrok]:https://ngrok.com
+[ngrok-k8s-gwapi-docs]:https://ngrok.com/docs/k8s/
+[ngrok-k8s-gwapi-qs]:https://ngrok.com/docs/k8s/
```

Comment:
These are both the same link, and neither seems to go to any Gateway API implementation information?

Reply:
There was originally a plan to have separate links, decided to switch to just one so I removed this reference.